2017-11-23 17:25:51,993 - log.py[line:34] - INFO: 测试开始时间：2017-11-23_17_25_51
2017-11-23 17:25:51,993 - server.py[line:18] - INFO: start /b taskkill /F /IM node.exe
2017-11-23 17:25:52,013 - server.py[line:20] - INFO: 启动appium服务
2017-11-23 17:25:52,013 - server.py[line:23] - INFO: start /b appium -a 127.0.0.1 -p 4723 --bootstrap-port 9517 --session-override --command-timeout 600
2017-11-23 17:25:52,043 - server.py[line:27] - INFO: http://localhost:4723/wd/hub/
2017-11-23 17:25:52,043 - connectionpool.py[line:208] - DEBUG: Starting new HTTP connection (1): 127.0.0.1
2017-11-23 17:25:54,086 - connectionpool.py[line:396] - DEBUG: http://127.0.0.1:8888 "GET http://localhost:4723/wd/hub/ HTTP/1.1" 404 47
2017-11-23 17:25:54,086 - server.py[line:35] - ERROR: 启动appium服务失败 exceptions must be old-style classes or derived from BaseException, not NoneType
2017-11-23 17:25:54,086 - log.py[line:88] - INFO:   start 初始化用例脚本.
